en/deploy/signup_pythonanywhere.md

```diff
@@ -1,13 +1,13 @@
 PythonAnywhere is a service for running Python code on servers "in the cloud". We'll use
 it for hosting our site, live and on the Internet.
 
-We will be hosting the blog we're building on PythonAnywhere. Sign up for a "Beginner" account on PythonAnywhere (the free tier is fine, you don't need a credit card).
+We will be hosting the blog we're building on PythonAnywhere. Sign up for a "Beginner" account on PythonAnywhere. (The free tier is fine, you don't need a credit card.)
```

Comment:
Another case where I think the parenthesis could also be dropped.

Reply:
I agree with dropping the parens.